README.md

```diff
@@ -1,33 +1,37 @@
 # AMP ⚡
 
-##### Stats:
```

Comment:
Do we want a "Badges" subheader here? No opinion either way just a thought as I look at these being removed.